query13.sql

```diff
@@ -6,4 +6,10 @@
 */
 
 -- Enter your SQL query here
-select ...
+select name station_name, id station_id, st_distance(
+  st_transform(st_setsrid(st_point(-75.1926,39.9523),4326),3857),
+  st_transform(the_geom, 3857)) distance_from_Meyerson
+  from indego_station_statuses
+  order by distance_from_Meyerson DESC
+  
```

Comment:
Note that "your query should only return one line", so you should use a `limit 1` on the last line.